src/app/screens/SignUp/styles.module.scss

```diff
@@ -50,4 +50,16 @@
     color: $earls-green;
     margin: 16px 24px 24px;
   }
+
+  .disabled {
+    background-color: #CCC;
```

Comment:
store this color on `src/scss/variables/_colors.tsx` remember styleguides
also use the color named given in `name-that-color` page